Library/Homebrew/test/formulary_spec.rb

```diff
@@ -55,37 +55,10 @@ def install
   end
 
   describe "::factory" do
-    before do
-      formula_path.dirname.mkpath
-      formula_path.write formula_content
-    end
-
-    it "returns a Formula" do
-      expect(described_class.factory(formula_name)).to be_a(Formula)
-    end
-
-    it "returns a Formula when given a fully qualified name" do
-      expect(described_class.factory("homebrew/core/#{formula_name}")).to be_a(Formula)
-    end
-
-    it "raises an error if the Formula cannot be found" do
-      expect do
-        described_class.factory("not_existed_formula")
-      end.to raise_error(FormulaUnavailableError)
-    end
-
-    it "raises an error if ref is nil" do
-      expect do
-        described_class.factory(nil)
-      end.to raise_error(TypeError)
-    end
-
-    context "with sharded Formula directory" do
-      let(:formula_name) { "testball_sharded" }
-      let(:formula_path) do
-        core_tap = CoreTap.instance
-        (core_tap.formula_dir/formula_name[0]).mkpath
-        core_tap.new_formula_path(formula_name)
+    context "without the API" do
```

Comment:
I added a new namespace here so that the before block which writes the formula source file locally doesn't get run before the API tests since it's not relevant and caused a false positive last time.